src/epsilon/main.d

```diff
@@ -61,6 +62,9 @@ void main(string[] args)
     {
         if (verbose)
             levels |= Level.trace;
+        else if (lsSupport)
```

Comment:
The idea was: _verbose_ would be requested by a human user only, never by the Epsilon LS. This should imply to get line/column positions with markers (^) instead of offset based positions. 
The verbose option would not work either for the Epsilon LS, the output parsing will fail.

Reply:
Does this make sense?

Reply:
There are two options:
1. language-server support does not work together with other switches
Then the application should fail with an error message.
2. language-server support could be used together with, for example `--verbose`
Perhaps an IDE supports CTRL-Click only with language-server offsets.
Then your solution would result in verbose output with offset positions.
What' s your preference?

Reply:
think the other switches should work. Only the `--verbose` is problematic. I'm not sure, if I always can skip the output when parsing.
Could we ensure the verbose output is always going to stdout?
Because the errors with offset position are always going to stderr, so I could ignore the verbose output very easily.
But I have to check, how verbose output behaves for the error parsing.

Reply:
... All output is going to stderr.
I have to check whether the output parsing is robust enough for that.

Reply:
Output parsing is robust enough. :smiley: 
And with commit c3aee25 we can just omit the special log level adjustment for `lsSupport` at all. I will remove it.